### PR TITLE
Milestone 4: Add Basic UI

### PR DESCRIPTION
**Pull request by Ibizugbe**
In this PR, I have completed the current milestone of the project and adhered to the given requirement.
- no linter error
- the project is professionally documented.
- created app.rb file to serve the console with a list of books, person, create a person, book, and rental.
- the main.rb file has a defined entry point that is a method called main and is invoked at the end of the file.
- it presents the user with a list of options to perform.
- it lets the user choose an option.
- there is a method to quit the app.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+      @people.each_with_index do |person, index|
+        puts "#{index}) Name: #{person.name} Age: #{person.age} Id: #{person.id}"
+      end
+
+      selected_person = gets.chomp.to_i
+
+      puts 'Enter date [YYYY-MM-DD]'
+      selected_date = gets.chomp.to_s
+
+      rental_item = Rental.new(selected_date, @people[selected_person], @books[selected_book])
```
Comment (rloterh) [inline app.rb:111]: - [ ] Your app is throwing an error in an attempt to create a rental. An error is thrown just after entering details to create a rental. Kindly resolve the same.

**Creating a rental error:**
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Abraham, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Highlights 👍  
- Console app starts correctly

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**You can also consider:**
- Kindly include in your ReadMe the _Getting Started_ section, which should include instructions on how to **setup** and **run** your application. You can follow this [Microverse recommended template](https://github.com/microverseinc/readme-template/blob/master/README.md).
- I think you should change the header title _Authors_ to _Author_ in your ReadMe, as this project is worked on by a sole contributor. 
- MIT license link in your ReadMe is broken, kindly endeavor to fix it.
Implementing the above change(s) will make your ReadMe appear more professional.
<br>
Cheers and Happy coding!👏 💻 🍷
<br><br>
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 1 above)
Comment (Ibizugbe) [reply app.rb:111]: Thanks a lot, @rloterh  for spotting this issue  I will do well to fix the issue.

**Review by DeliceLydia — APPROVED** [review]
## Hi @Ibizugbe, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ibizugbe** [comment]
Thanks, @DeliceLydia for approving the pr

**Pull request merged**